code/landing-page/package.json

```diff
@@ -18,7 +18,9 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "install": "npm install",
+    "frontend": "npm run install && npm run start"
```

Comment:
Is `npm install` and `npm frontend` necessary? The deployment script can do `npm i && npm start`.